bucket/qtdesigner.json

```diff
@@ -0,0 +1,36 @@
+{
+    "version": "5.14.1",
+    "description": "Qt tool for designing and building GUIs with Qt Widgets",
+    "homepage": "https://doc.qt.io/qt-5/qtdesigner-manual.html",
+    "license": "GPL-3.0",
+    "url": "https://files.pythonhosted.org/packages/3c/c5/f010b4d1f39993fefeb124df98f7b279f1a0339179f998976a5ab2e8da15/PyQt5Designer-5.14.1-py3-none-win32.whl",
+    "hash": "9d22b4f0f47d81859f253850f2a2d66e29011b4a380541b6a891e035a0914769",
+    "installer": {
+        "script": "Expand-7zipArchive $dir\\PyQt5Designer-$version-py3-none-win32.whl -ExtractDir QtDesigner -Removal"
+    },
+    "shortcuts": [
+        [
+            "designer.exe",
+            "Qt Designer\\Qt Designer"
+        ],
+        [
+            "linguist.exe",
+            "Qt Designer\\Qt Linguist"
+        ]
+    ],
+    "bin": "qml.exe",
+    "checkver": {
+        "url": "https://pypi.org/pypi/PyQt5Designer/json",
+        "jsonpath": "$.urls[?(@.url =~ /.*win32\\.whl/')].url",
+        "regex": "^https://(.*?PyQt5Designer-([\\d\\.]*?)-.*?)$",
+        "replace": "$2"
```

Comment:
Find more relevant source

Reply:
That is the source. They package the tool and release it via PyPI.

Reply:
> They package the tool and release it via PyPI.

Source of this claim?? There is no such information in docs